app/src/main/res/values/colors.xml

```diff
@@ -2,6 +2,8 @@
 <resources>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="light_grey">#F7F7F7</color>
```

Comment:
CI warnings says : The resource `R.color.light_grey` appears to be unused.

Therefore you should remove it, as well as some other colors

Reply:
Those colors are part of a well thought theme I made and could be used at anytime in the project. I think that we should keep them until we finish working on a proper day and night theme

Reply:
Good idea !